agents/schemas/profile.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "agents/Profile",
+  "title": "Profile",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "integer",
+      "description": "Id referencing profile"
+    },
+    "agentId": {
+      "type": "integer",
```

Comment:
can this be

```
{
  "$ref": "/agents/Agent#/definitions/id"
}
```

oh wait nevermind, we don't have a schema for Agent. all good!